ru-ru/git-ru.html.markdown

```diff
@@ -0,0 +1,443 @@
+# GIT
```

Comment:
Please add the metadata. Put your name into the `translators` section (please see other Russian articles in the `ru-ru` folder.